support/oidc-discovery-provider/cert_manager_test.go

```diff
@@ -0,0 +1,423 @@
+package main
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/spiffe/spire/pkg/common/x509util"
+	"github.com/spiffe/spire/test/testkey"
+	"github.com/stretchr/testify/require"
+)
+
+var logger, logHook = test.NewNullLogger()
+
+func TestTLSConfig(t *testing.T) {
+	oidcServerKey := testkey.MustEC256()
```

Comment:
nit: it's ok here, but usually i try not to use `testkey` package level functions except at the package scope. If somebody runs `go test -count=1000`, it can end up auto-generating a ton of keys :)

My suggestion would be to move this key var to the package scope